client/my-sites/site-settings/media-settings/carousel-settings.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+const CarouselSettings = props => {
+	const handleCheckbox = ( event ) => {
+		props.onInputChange( {
+			[ event.target.name ]: event.target.checked
+		} );
+	};
+
+	const handleSelect = ( event ) => {
+		props.onInputChange( {
+			[ event.target.name ]: event.target.value
+		} );
+	};
+
+	return (
+		<div className="media-settings__module-settings is-indented">
+			<FormLabel>
+				<FormCheckbox
+					name="carousel_display_exif"
+					checked={ props.carousel_display_exif || false }
+					onChange={ handleCheckbox }
+					disabled={ props.submittingForm } />
+				<span>{ props.translate( 'Show photo metadata (Exif) in carousel, when available' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Show photo metadata (<a href="http://en.wikipedia.org/wiki/Exchangeable_image_file_format" target="_blank">Exif</a>) in carousel, when available.' )` *ES Score: 7.18*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-site-settings-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).